src/apps/referrals/apps/send-referral/client/SendReferralConfirmation.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { SummaryTable, FormActions } from 'data-hub-components'
+import { H4, Button, Link } from 'govuk-react'
+import SecondaryButton from '../../../../../client/components/SecondaryButton'
+
+const StyledParagraph = styled('p')`
+  font-size: 16px;
+`
+
+const SendReferralConfirmation = ({
+  companyName,
+  companyId,
+  adviser,
+  subject,
+  contact,
+  notes,
+  cancelUrl,
+  onBack,
+  csrfToken,
+}) => {
+  return (
+    <>
+      <SummaryTable caption="Check referral details">
+        <SummaryTable.Row heading="Send this company record to">
+          {adviser.name}
+        </SummaryTable.Row>
+        <SummaryTable.Row heading="Subject">{subject}</SummaryTable.Row>
+        <SummaryTable.Row heading="Notes">
+          {notes || 'No notes added'}
+        </SummaryTable.Row>
+        <SummaryTable.Row heading="Company contact">
+          {contact?.name || 'No contact added'}
+        </SummaryTable.Row>
+      </SummaryTable>
+      <SecondaryButton onClick={onBack}>Edit referral</SecondaryButton>
+      <H4>What happens next</H4>
+      <StyledParagraph>
+        <p>
+          <p>
+            Clicking “Send referral” will show the referral in the activity of{' '}
+            {` ${companyName}`}, as well as in the Referrals section on both
```

Comment:
I think we will have two empty spaces here, i.e.
```
Clicking “Send referral” will show the referral in the activity of__{companyName}
```

```suggestion
            Clicking “Send referral” will show the referral in the activity of{' '}
            {companyName}, as well as in the Referrals section on both
```

Reply:
Good catch! Have pushed the change :)